.infrastructure/hpa.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: todoapp
+  namespace: mateapp
```

Comment:
The namespace is set to 'mateapp', which is inconsistent with the intended 'todoapp'. Please correct this to ensure consistency across your configuration.